### ISSUE TITLE
Focus indicator on Skip link in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10181]
## Description
In the CMS, the skip link does not have a visible focus indicator.

## Additional Context
If the skip link could be updated to be more prominent it would assist users in noticing this when it was focused on.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184215965-a3be6a20-80f0-450b-849b-72ba65d5f08e.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 2.4.7](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)

## Acceptance Criteria
- [ ] Interactive design to determine pattern for skip link
- [ ] Technical review
- [ ] Change management consulted
- [ ] Implementation ticket created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by srancour** [comment]
This is something that still exists and should still be worked on.
